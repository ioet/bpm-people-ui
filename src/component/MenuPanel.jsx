import React from 'react';
import MenuItem from './MenuItem';
import styles from '../generalStyle.css';

const MenuPanel = () => (
  <div className={styles.menuPanelStyle}>
    <MenuItem text="People" />
    <MenuItem text="Projects" />
    <MenuItem text="Skills" />
  </div>
);

export default MenuPanel;
