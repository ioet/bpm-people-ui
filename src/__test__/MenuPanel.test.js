import React from 'react';
import MenuPanel from '../component/MenuPanel';


describe('<MenuPanel/>', () => {
  test('Renders correctly with children', () => {
    const menuPanelWrapper = shallow(<MenuPanel />);

    expect(toJson(menuPanelWrapper)).toMatchSnapshot();

    expect(menuPanelWrapper.find('MenuItem').length).toEqual(3);
    expect(menuPanelWrapper.find('div.menuPanelStyle').length).toEqual(1);
  });
});
