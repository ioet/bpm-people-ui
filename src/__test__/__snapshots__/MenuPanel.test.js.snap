// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuPanel/> Renders correctly with children 1`] = `
<div
  className="menuPanelStyle"
>
  <MenuItem
    text="People"
  />
  <MenuItem
    text="Projects"
  />
  <MenuItem
    text="Skills"
  />
</div>
`;
